contrib/completions/bash/buildah

```diff
@@ -396,11 +410,23 @@ return 1
   "
 
      local options_with_args="
+     --add-host
+     -c
```

Comment:
can you move this to the cpu-shares line?

Reply:
fixed